cmd/syft/cli/options/pkg.go

```diff
@@ -1,23 +1,18 @@
 package options
 
-import (
-	"github.com/anchore/syft/syft/pkg/cataloger"
-)
+import "github.com/anchore/syft/syft/cataloging"
 
-type pkg struct {
-	Cataloger               scope `yaml:"cataloger" json:"cataloger" mapstructure:"cataloger"`
-	SearchUnindexedArchives bool  `yaml:"search-unindexed-archives" json:"search-unindexed-archives" mapstructure:"search-unindexed-archives"`
-	SearchIndexedArchives   bool  `yaml:"search-indexed-archives" json:"search-indexed-archives" mapstructure:"search-indexed-archives"`
+type packageConfig struct {
+	SearchUnindexedArchives         bool `yaml:"search-unindexed-archives" json:"search-unindexed-archives" mapstructure:"search-unindexed-archives"`
+	SearchIndexedArchives           bool `yaml:"search-indexed-archives" json:"search-indexed-archives" mapstructure:"search-indexed-archives"`
+	ExcludeBinaryOverlapByOwnership bool `yaml:"exclude-binary-overlap-by-ownership" json:"exclude-binary-overlap-by-ownership" mapstructure:"exclude-binary-overlap-by-ownership"` // exclude synthetic binary packages owned by os package files
 }
 
-func defaultPkg() pkg {
-	c := cataloger.DefaultSearchConfig()
-	return pkg{
-		SearchIndexedArchives:   c.IncludeIndexedArchives,
-		SearchUnindexedArchives: c.IncludeUnindexedArchives,
-		Cataloger: scope{
-			Enabled: true,
-			Scope:   c.Scope.String(),
-		},
+func defaultPkg() packageConfig {
+	c := cataloging.DefaultArchiveSearchConfig()
+	return packageConfig{
+		SearchIndexedArchives:           c.IncludeIndexedArchives,
+		SearchUnindexedArchives:         c.IncludeUnindexedArchives,
+		ExcludeBinaryOverlapByOwnership: true,
```

Comment:
**open question**: this logically affects packages, so belongs in the package config. However, in terms of when it's needed, this is really used after packages are created and uses the relationships to determine how to prune packages (so functionally belongs in the relationships config).

These two things are at odds and are also at odds in app config vs API config.

What do folks think about this?

Reply:
I thought this was already in the relationships config? I'm missing something.

Reply:
there is a CLI-level configuration and an API-level configuration -- currently this option is on the Packages config on the CLI-level configuration and the Relationships config in the API-configuration. There seems to be logic to reason about it either way. Consistency is better than correctness in my opinion (if this is a situation where they are really mutually exclusive... but I'm not certain if that's true). Looking for options here.

Reply:
we'll leave this as is